config/build/remove-test-types-from-dist.ts

```diff
@@ -0,0 +1,38 @@
+import * as fs from "fs";
+import * as path from "path";
+
+import * as fglob from "fast-glob";
+
+const rootDir = path.join(__dirname, "..");
```

Comment:
isn't the root dir 2 directories up from here?

Reply:
Thanks for catching this.  Thankfully we don't need this file any more.